R/plot.R

```diff
@@ -70,6 +74,9 @@ elic_plot <- function(x,
                       round,
                       var,
                       ...,
+                      xlab = var,
+                      ylab = "Expert",
+                      title = paste("Round", round),
```

Comment:
Please, move them to the bottom:

* line_width
* title
* xlab
* ylab
* family